charts/coredns/Chart.yaml

```
@@ -19,5 +19,5 @@ engine: gotpl
 type: application
 annotations:
   artifacthub.io/changes: |
-    - kind: added
-      description: Added autoscaler podAnnotations
+    - kind: fixed
+      description: Wrong namespaceSelector is ServiceMonitor when use custom monitoring namespace
```

Small nit, just to make the wording more clear

```suggestion
      description: Fix wrong namespaceSelector in ServiceMonitor when using custom monitoring namespace
```